plugins/aws-s3-storage/src/index.ts

```diff
@@ -31,7 +31,8 @@ export default class S3Database implements IPluginStorage<S3Config> {
     if (!this.config.bucket) {
       throw new Error('s3 storage requires a bucket');
     }
-    const configKeyPrefix = this.config.keyPrefix;
+    this.config.bucket = process.env[this.config.bucket] || this.config.bucket;
```

Comment:
Could you provide some examples or how the users might use this ? to have a better overview of your solution.

Reply:
## Example 1:

``` yaml
store:
  aws-s3-storage:
    bucket: S3_BUCKET
    keyPrefix: someConstantPrefix
    region: S3_BUCKET_REGION
```
Given the following environment variables are set on the host running Verdaccio:

- `S3_BUCKET=myBucket`
- `S3_BUCKET_REGION=some-aws-region`

This would result in the plugin initialization with the following values:
``` json
{
  "bucket": "myBucket",
  "keyPrefix": "someConstantPrefix",
  "region": "some-aws-region"
}
```

## Example 2:

``` yaml
store:
  aws-s3-storage:
    bucket: bucketName
    keyPrefix: someConstantPrefix
    region: eu-west-1
```
Given there are NO environment variables are set with names `bucketName`, `someConstantPrefix` and `eu-west-1`:

This would result in the plugin initialization with the following values:
``` json
{
  "bucket": "bucketName",
  "keyPrefix": "someConstantPrefix",
  "region": "eu-west-1"
}
```

## Example 3

AWS Access keys would have similar behaviour with the addition of:
if no environment variables are set and there are also no values provided in the config.yaml file, then it falls back using the instance/container's IAM role.

Reply:
Great ! I like it. Could you rebase from master and add this examples to the README? I think will have more visibility.